ui-tests/src/test/resources/features/other/integration-metrics.feature

```diff
@@ -234,4 +243,6 @@ Feature: Metrics
     And check that last processed date is valid
     And check that number of valid messages is 1
     And check that number of error messages is 0
-    And check that number of total messages is 1
\ No newline at end of file
+    And check that number of total messages is 1
+    And check that uptime for webhook-to-db pod is valid
+    And check that startdate for webhook-to-db pod is valid
```

Comment:
please add a new line here, otherwise it's ok

Reply:
added